test/source/patterns.ts

```diff
@@ -98,10 +99,26 @@ for (const buildType of ['chrome-consumer', 'chrome-enterprise']) {
       }
     }
   }
-  const gmailCs = manifest.content_scripts?.find(cs => cs.matches?.includes('https://mail.google.com/*'));
-  if (!gmailCs?.css?.length || !gmailCs.js?.length) {
-    console.error(`Missing content_scripts declaration for Gmail in ${buildType}/manifest.json`);
-    errsFound++;
+  if (buildType === 'thunderbird-consumer') {
+    if (manifest.manifest_version !== 2) {
+      console.error(`${buildType} - Manifest version is not 2`);
+      errsFound++;
+    }
+    if (!Array.isArray(manifest.web_accessible_resources)) {
+      console.error(`${buildType} - web_accessible_resources should be an array`);
+      errsFound++;
+    }
+    if (typeof manifest.content_security_policy !== 'string') {
+      console.error(`${buildType} - content_security_policy should be a string`);
+      errsFound++;
+    }
+  }
+  if (webBrowserBuilds) {
+    const gmailCs = manifest.content_scripts?.find(cs => cs.matches?.includes('https://mail.google.com/*'));
+    if (!gmailCs?.css?.length || !gmailCs.js?.length) {
+      console.error(`Missing content_scripts declaration for Gmail in ${buildType}/manifest.json`);
+      errsFound++;
+    }
```

Comment:
after the latest change for `content_scripts`, I think this test should be applied to thunderbird builds too

Reply:
Thanks for the tip @sosnovsky. I made a little refactor to it as I think it should be described as generic since most builds that will come will likely to be tested for this content_script contents.